plugins/coverbrowser.koplugin/bookinfomanager.lua

```diff
@@ -237,7 +249,8 @@ function BookInfoManager:getSetting(key)
     return self.settings[key]
 end
 
-function BookInfoManager:saveSetting(key, value)
+function BookInfoManager:saveSetting(key, value, reload)
+    reload = reload or true
```

Comment:
Haven't noticed it before :)
If you provide `reload=false`, you'll end up with `reload = false or true`, so with `true`.
(The right way is: `if reload == nil then reload = true end` instead.)

If you haven't noticed any thing strange since then :) I suggest we drop that reload parameter, and always end up with loadSettings, as before, to keep things simple.
(As you'd use it only when migrating, yes, you'd do 2 or 3 unneeded saves, but that's a one time thing when migrating the old setting.)

Reply:
Totally missed the `reload=false` implication :(
If we reload settings on every time, the order in which we reload can impact the correctness of the translation. So I'd rather go with no translation at all, and the old settings will be kept lying around.

Reply:
Oh, right, indeed.
Well, I'm fine with both options: no migration, or just fixing your `reload = false or true`, so do as you wish